### ISSUE TITLE
Missing screw pegs

### ISSUE THREAD

**Issue by Aask42** [So11Deo6loria/bhvBadge2023#3]
I'm experimenting with printing the heart in various translucent materials, and noticed the model of the heart is missing the peg-stands for mounting to the PCB. Any chance we could get an update that has a version of the heart with the screw holes in addition to this one? I can make one but if one already exists it just makes sense to aask :):

https://github.com/So11Deo6loria/bhvBadge2023/blob/main/models/HeartModel.tar.gz

<img width="817" alt="image" src="https://github.com/So11Deo6loria/bhvBadge2023/assets/13933200/388c3e53-109e-4d0d-8bca-766edb140dff">

**Comment by heyfryman** [comment]
> I'm experimenting with printing the heart in various translucent materials, and noticed the model of the heart is missing the peg-stands for mounting to the PCB. Any chance we could get an update that has a version of the heart with the screw holes in addition to this one? I can make one but if one already exists it just makes sense to aask :):
> 
> https://github.com/So11Deo6loria/bhvBadge2023/blob/main/models/HeartModel.tar.gz
> 
> <img alt="image" width="817" src="https://user-images.githubusercontent.com/13933200/263062987-388c3e53-109e-4d0d-8bca-766edb140dff.png">

Apologies for not uploading the version of the heart with the holes for the screws and potentially the gasket STL for mounting the heart on the PCB.  I'll get that uploaded this afternoon.

**Comment by heyfryman** [comment]
@Aask42 
[BHV_Heart_Model.zip](https://github.com/So11Deo6loria/bhvBadge2023/files/12439606/BHV_Heart_Model.zip)

Attached is the heart model with the cavities for the screws. Unfortunately it is not a single object, but several objects that I still need to get merged into a single object. Once I'm able to get this merged into a single object, I'll make a post here.

### Important info about the model:
1. This is PrusaSlicer (v2.6.0) 3mf file, which also contains the configuration information for a printer.
2. When you import the file into your slicer, you want to "Import 3D models only" ![image](https://github.com/So11Deo6loria/bhvBadge2023/assets/27772193/55b213ee-a093-445a-ae99-8d0d9ed72cf5)
3. Select "OK"
4. Make sure you still have the correct printer and filament settings

Now you should be able to print the model correctly.

### Other info about the badge model:
1. The screw cavities are designed for nylon 2.5M (~2.5mm diameter) screws that are 10mm in length.
2. When using TPU or other flexible material, we recommend using using a rod/screwdriver/Allen key or something similar to push into each cavity before inserting the screws to ensure it is clear of debris or any filament that could obstruct the path. If you print the modeling using non-flexible material you should tap each cavity with a 2.5M tap.
3.  Infill: We used 10% infill in a Lightning pattern, but we tested it with several different infills and they all worked. We chose Lightning at 10% infill because it prevented the light from calumniating.

Good luck and please share your results!

**Comment by Aask42** [comment]
Getting this from prusaslicer to my ankermake has been a journey but we got it going! Will definitely show results of everything we try out once the print is complete in a few hours. 

<img width="591" alt="image" src="https://github.com/So11Deo6loria/bhvBadge2023/assets/13933200/44cd13ac-6c43-4f57-8f8a-e792a24db1cd">

**Comment by Aask42** [comment]
Oh before I foget, another note, the original model up here is also the wrong version of the file for fitting the final PCB: 
<img width="963" alt="image" src="https://github.com/So11Deo6loria/bhvBadge2023/assets/13933200/7626a58e-9fc0-4a7f-b605-f75242c47f48">

**Comment by heyfryman** [comment]
Thank you for information - we will get that model removed.

**Comment by Aask42** [comment]
It looks great! Thank you for your help!

![red_heart](https://github.com/So11Deo6loria/bhvBadge2023/assets/13933200/198df7be-8803-4475-90ba-811e88b7264b)

<img width="475" alt="image" src="https://github.com/So11Deo6loria/bhvBadge2023/assets/13933200/c22d398c-61f3-444e-8ed1-484138d2ef41">
